.gitignore

```diff
@@ -12,3 +12,5 @@ notifier.json
 /ereb/etc
 /ereb/var
 /ereb.egg-info/*
+.env/
```

Comment:
In public repos it's better to use your own ignore list [http://stackoverflow.com/questions/653454/how-do-you-make-git-ignore-files-without-using-gitignore](http://stackoverflow.com/questions/653454/how-do-you-make-git-ignore-files-without-using-gitignore)